src/back/handlers.py

```diff
@@ -0,0 +1,30 @@
+"""Module containing handlers."""
+import dataclasses
+from datetime import datetime
+from typing import Self
+
+from src.back.interfaces.handlers import MessageHandlerInterface
+from src.back.interfaces.io import WriteStreamInterface
+from src.back.interfaces.message import MessageInterface
+from src.back.interfaces.room_registry import RoomRegistryInterface
+from src.back.room import Room
+
+
+@dataclasses.dataclass
+class CreateRoomMessage(MessageInterface):
+    """Create room message."""
+
+    name: str
+    created_at: datetime
+
+
+@dataclasses.dataclass
+class CreateRoomHandler(MessageHandlerInterface[CreateRoomMessage]):
+    """Interface for create room."""
+
+    room_registry: RoomRegistryInterface
+
+    def handle(self: Self, message: CreateRoomMessage, stream: WriteStreamInterface) -> None:
```

Comment:
5) And thus, I am now able to use CreateRoomMessage as a type for message argument

Reply:
Yes